pages/auth/signup.tsx

```diff
@@ -141,52 +141,57 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
   const token = asStringOrNull(ctx.query.token);
   if (!token) {
     return {
-      notFound: true,
+      props: {
+        isGoogleLoginEnabled: IS_GOOGLE_LOGIN_ENABLED,
+        isSAMLLoginEnabled,
+        trpcState: ssr.dehydrate(),
+      },
     };
-  }
-  const verificationRequest = await prisma.verificationRequest.findUnique({
-    where: {
-      token,
-    },
-  });
+  } else {
```

Comment:
This `else` statement can be removed because there is a `return` inside `if (!token)`.